src/Header.js

```diff
@@ -20,26 +19,32 @@ class Header extends Component {
     }
 
     render() {
-
         const loggedIn = isLoggedIn();
+        const currentUser = currentUserName();
         return (
         <header className="navbar">
             <section className="navbar-section">
                 <NavLink className="btn btn-link" to="/">Home</NavLink>
                 <NavLink className="btn btn-link" to="/new">New Game</NavLink>
                 <NavLink className="btn btn-link" to="/games">Games</NavLink>
                 <NavLink className="btn btn-link" to="/notifications">Notifications</NavLink>
+                {loggedIn &&
+                    <NavLink className="btn btn-link" to="/my_page">My page</NavLink>
```

Comment:
Should we move this to the right side of the bar, like the rest of the login-related stuff?